src/PhpCodeSnifferRunner/Runner.php

```diff
@@ -17,7 +17,7 @@ class Runner
 {
     public function run(string $basePath): Report
     {
-        $cliCommand = sprintf('vendor/bin/phpcs -q --report=json --basepath=%s', escapeshellarg($basePath));
+        $cliCommand = sprintf($basePath.'/vendor/bin/phpcs -q --report=json');
```

Comment:
Prepending the base path is a good addition. However, I prefer string templating via `sprintf()` instead of concatenation. Could you change that?

The `--basepath` argument should not be omitted. This arguments makes sure that the report contains file paths relative to the base path instead of absolute paths.